content/flux/v0.x/join-data/time.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Join on time
+description: >
+  Use [`join.time()`](/flux/v0.x/stdlib/join/time/) to join two streams of data
+  based on time values in the `_time` column.
+  This type of join operation is common when joining two streams of
+  [time series data](/influxdb/latest/reference/glossary/#time-series-data).
+menu:
+  flux_0_x:
+    parent: Join data
+weight: 104
+related:
+  - /flux/v0.x/stdlib/join/
+  - /flux/v0.x/stdlib/join/time/
+list_code_example: |
+  ```js
+  import "join"
+
+  left = from(bucket: "example-bucket-1") |> //...
+  right = from(bucket: "example-bucket-2") |> //...
+
+  join.time(
+      left: left,
+      right: right,
+      as: (l, r) => ({l with field1: l._value, field2: r._value_}),
+  )
+  ```
+---
+
+Use [`join.time()`](/flux/v0.x/stdlib/join/time/) to join two streams of data
+based on time values in the `_time` column.
+This type of join operation is common when joining two streams of
+[time series data](/influxdb/latest/reference/glossary/#time-series-data).
+
+`join.time()` can use any of the available join methods.
+Which method you use depends on your desired behavior:
+
+- **inner** _(Default)_:
+  Drop any rows from both input streams that do not have a matching
+  row in the other stream.
+
+- **left**:
+  Output a row for each row in the **left** data stream with data matching
+  from the **right** data stream. If there is no matching data in the **right**
+  data stream, columns with values from the **right** data stream are _null_.
+
+- **right**: 
+  Output a row for each row in the **right** data stream with data matching
+  from the **left** data stream. If there is no matching data in the **left**
+  data stream, columns with values from the **left** data stream are _null_.
```

Comment:
Not sure how to word it, but we should clarify that only _non-group key_ columns from the left data stream are null.